.github/workflows/e2e.yml

```diff
@@ -49,8 +49,5 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Install buf
-        uses: bufbuild/buf-setup-action@v1.36.0
-
       - name: Run E2E Tests
```

Comment:
Just to confirm my understanding, this actually wasn't being used anymore here, since we removed the buf usage from the build script, right?